src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
```suggestion
public class ColorSupplier {
    private Random random = new Random();
    
    String getRandomColor() {
        int index = random.nextInt(Color.values().length);
```
lets me random global